### PR TITLE
add Ping to ICP Showcase

### PR DESCRIPTION
**Pull request by Saumay-Agrawal**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

**Comment by sa-github-api** [comment]
Dear @Saumay-Agrawal,

In order to potentially merge your code in this open-source repository and therefore proceed with your contribution, we need to have your approval on DFINITY's [CLA](https://github.com/dfinity/cla/blob/main/CLA.md).

If you decide to agree with it, please visit this [issue](https://github.com/dfinity/cla/issues/422) and read the instructions there. Once you have signed it, re-trigger the workflow on this PR to see if your code can be merged.

— The DFINITY Foundation

File: showcase.json [hunk 1]
```diff
@@ -3228,6 +3228,26 @@
   "twitter": "https://twitter.com/ICLighthouse",
   "youtube": "https://youtu.be/Z92o1oCH598",
   "submittableId": "41203767"
+},
+{
+  "name": "Ping",
+  "description": "Ping is a notification service that boosts ecosystem engagement by creating a direct communication channel between dapp developers and their respective users.",
+  "website": "https://h7jna-pqaaa-aaaak-afgiq-cai.icp0.io/",
```
Comment (jennifertrin) [inline showcase.json:3235]: @Saumay-Agrawal Can you please let us know if Ping is live yet?

File: showcase.json [hunk 2]
```diff
@@ -3228,6 +3228,26 @@
...
+  "logo": "/img/showcase/ping_logo.webp",
+  "screenshots": [],
```
Comment (jennifertrin) [inline showcase.json:3237]: Can you please remove any key-value pairs that do not have a value?

**Review by jennifertrin — CHANGES_REQUESTED** [review]
Can you please review the CLA steps above as well?

File: showcase.json (hunk 1 above)
Comment (Saumay-Agrawal) [reply showcase.json:3235]: @jennifertrin yes, the application is live on ICP mainnet at this link. Let me know if it isn't working for you. I'll look into it.

File: showcase.json (hunk 2 above)
Comment (Saumay-Agrawal) [reply showcase.json]: Sure, fixed the file.

File: showcase.json (hunk 1 above)
Comment (jennifertrin) [reply showcase.json:3235]: > @jennifertrin yes, the application is live on ICP mainnet at this link. Let me know if it isn't working for you. I'll look into it.

It takes about a minute to login. Are you running any other functions when a user logs in that is causing the slow speed?

**Review by jennifertrin — APPROVED** [review]

**Comment by jennifertrin** [comment]
@Saumay-Agrawal Can you please resolve merge conflicts or allow us to edit the repo?

**Comment by Saumay-Agrawal** [comment]
@jennifertrin I've resolved the merge conflicts.

**Review by jennifertrin — APPROVED** [review]

**Pull request merged**